Demo/Demo/DemoSettings/DemoSettings.swift

```diff
@@ -32,13 +33,14 @@ enum DemoSettings {
         }
     }
 
-    static var clientID: String {
-        switch environment {
-        case .sandbox:
-            return "AcXwOk3dof7NCNcriyS8RVh5q39ozvdWUF9oHPrWqfyrDS4AwVdKe32Axuk2ADo6rI_31Vv6MGgOyzRt"
-        case .live:
-            // TODO: Investigate getting a prod testing account that doesn't charge real money
-            return "TODO"
+    static var merchantIntegration: MerchantIntegration {
+        get {
+            UserDefaults.standard.string(
+                forKey: MerchantIntegrationDefaultKey)
+            .flatMap { MerchantIntegration(rawValue: $0) } ?? .unspecified
```

Comment:
We could rename `.unspecified` to `.default`. This is what we do on Android and the sample merchant server also has this concept [here](https://github.paypal.com/Braintree/sdk-sample-merchant-server/blob/main/lib/routers/pay-pal.js#L195).

Reply:
Good call! I was going to look at Android implementation to see how you handled our old default one.

Reply:
As we discussed, ".default" caused problem with compiler because it is a reserved word in Swift.

Reply:
Yeah you would have to escape default since it's protected. On the BT side we use `case none` ([example here](https://github.com/braintree/braintree_ios/blob/e70b7f2c52177c25362bc242fdaa5f82e02a16e1/Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift#L41-L56)) so maybe we should use the same across both SDKs?

Reply:
+1 for `.none` that's a good call. We could update on Android as well.

Reply:
Addressed here: c92f46627ba03080f4ee7ffca23064d84ad6a4f5 
I did make .direct as default selection in the picker and also in parameter for building order request
at Ryan's suggestion. They're both direct merchant credentials. So "none" maps to our original value 
for direct merchant that is accessible without specifying a specific path to the sample merchant server.